server/src/main/java/org/apache/gravitino/server/web/Utils.java

```diff
@@ -122,6 +122,28 @@ public static Response alreadyExists(String type, String message, Throwable thro
         .build();
   }
 
+  public static Response nonInUse(String message, Throwable throwable) {
+    return nonInUse(throwable.getClass().getSimpleName(), message, throwable);
+  }
+
+  public static Response nonInUse(String type, String message, Throwable throwable) {
+    return Response.status(Response.Status.CONFLICT)
+        .entity(ErrorResponse.nonInUse(type, message, throwable))
+        .type(MediaType.APPLICATION_JSON)
+        .build();
+  }
+
+  public static Response inUse(String message, Throwable throwable) {
+    return inUse(throwable.getClass().getSimpleName(), message, throwable);
+  }
+
+  public static Response inUse(String type, String message, Throwable throwable) {
+    return Response.status(Response.Status.CONFLICT)
```

Comment:
Maybe I prefer 503. Just a discussion not a suggestion.

Reply:
`SERVICE_UNAVAILABLE(503, "Service Unavailable")` means the server status, not the resource. So I think it's not suitable.